app/models/acl_dao.rb

```diff
@@ -93,7 +93,12 @@ def send_request_to_cmr(action, endpoint, data = nil)
     msg += "#{@base_url}/#{endpoint}, response.status=#{response.status}, "
     msg += "body=#{response.body}"
     Rails.logger.info(msg)
-    json = JSON.parse(response.body)
+    json = {}
+    begin
+      json = JSON.parse(response.body)
+    rescue JSON::ParserError
+      Rails.logger.info "Error parsing JSON response from CMR retrieving ACLs.  response=#{response.body}"
+    end
```

Comment:
Are the callers of this prepared to handle and understand the empty json?  Is it possible they're proceeding with a failure silently rather than dying because they received an irreconcilable parse error?

Reply:
Well this is a one-off, but I see your point.  I have another error message I can use instead that is a bit more generic.

Reply:
Oh wait, they actually don't see this error message, that is what goes to Rails logger, this should forces it to return {} which the calling class parses and presents an error since it got nothing legible from CMR.

Reply:
That's more of what I was trying to ask about.  It seemed like the old way was failing loudly when it broke and this way had the potential to be failing quietly, but you say it's covered.